uber/utils.py

```diff
@@ -1060,10 +1060,10 @@ def charge_cc(self, session, token):
         else:
             if self.models:
                 self._stripe_transaction = self.stripe_transaction_from_charge()
+                session.add(self._stripe_transaction)
                 for model in self.models:
                     multi = len(self.models) > 1
                     session.add(self.stripe_transaction_for_model(model, self._stripe_transaction, multi))
-                session.add(self._stripe_transaction)
```

Comment:
Out of curiosity, why was this moved?  It's not obvious to me why it would make a difference.

Reply:
Oh, I at first had no idea what was happening with the session, so I moved this just to see if it changed anything. I don't think it does, though.